precompiles/src/precompile_set.rs

```diff
@@ -833,6 +834,68 @@ impl<A> IsActivePrecompile for RevertPrecompile<A> {
 	}
 }
 
+/// Precompiles that were removed from a precompile set.
+/// Still considered precompiles but are inactive and always revert.
+pub struct RemovedPrecompilesAt<A>(PhantomData<A>);
```

Comment:
This is alread included in the master. https://github.com/polkadot-evm/frontier/blob/master/precompiles/src/precompile_set.rs#L836-L882

Reply:
Fine, it's for multiple precompiles.